src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -1122,13 +1147,13 @@ PaError OpenStream( struct PaUtilHostApiRepresentation *hostApi,
         stream->inputDevice = inputParameters->device;
 
         /* Convert positive suggestedLatency from seconds to microseconds, otherwise default to zero. */
-        if (inputParameters->suggestedLatency >= 0)
+        if ( inputParameters->suggestedLatency > 0.0 )
         {
             stream->suggestedLatencyUSecs = (unsigned int) (inputParameters->suggestedLatency * 1e6 + 1.0f);
         }
         else
         {
-            stream->suggestedLatencyUSecs = 0;
+            stream->suggestedLatencyUSecs = (unsigned int) (PA_PULSEAUDIO_DEFAULT_MIN_LATENCY * 1e6 + 1.0f);
```

Comment:
`inputParameters->suggestedLatency` of `0` should result in the same buffer size as suggested latency `0 + epsilon`. So maybe just set this to 1usec here if 0 isn't working.

Reply:
Ok this in reverted now. Sorry I didn't understand the epsilon thing.